values/wire-server/prod-values.example.yaml

```diff
@@ -4,6 +4,7 @@ tags:
   account-pages: true
   team-settings: true # enable if you need team-settings. Requires a pullSecret
   legalhold: false # Enable if you need legalhold
+  federator: false # Enable to use federation
```

Comment:
Damn: This breaks the bundling of the `federator` image, because this file is used in https://github.com/wireapp/wire-server-deploy/blob/master/nix/scripts/list-helm-containers.sh#L34